dependencies.tsv

```diff
@@ -13,6 +13,7 @@ github.com/testing-cabal/subunit-go	git	00b258565a5cf3adaa24b68d31c9e6ec3d2cdbe7
 golang.org/x/crypto	git	60052bd85f2d91293457e8811b0cf26b773de469	2015-06-22T23:34:07Z
 golang.org/x/net	git	3b90a77d2885fb0429e8a21ab72fc73ca6f8b401	2016-01-05T05:09:10Z
 gopkg.in/check.v1	git	64131543e7896d5bcc6bd5a76287eb75ea96c673	2014-10-24T13:38:53Z
+gopkg.in/macaroon.v1	git	ab3940c6c16510a850e1c2dd628b919f0f3f1464	2015-01-21T11:42:31Z
```

Comment:
Hmm... it looks like there is no package for macaroon.v1, what should I do in that case?